CONTRIBUTOR-LADDER.md

```diff
@@ -150,3 +150,13 @@ Inactive members are defined as members of the Cilium org with **no** contributi
 
 **Note:** Devstats does not take into account non-code contributions. If a non-code contributing member is accidentally removed this way, they may open an issue to quickly be re-instated.
 
+## General guidance on making contributions and expectations
+
+We welcome people to raise PRs, issues and CFPs in the Cilium project. If you're new to the project we'd strongly encourage you to start off with small, but meaningful, changes first, as these are easier and quicker for committers / reviewers to assess and give feedback on. You'll get the opportunity to learn about the [process and expectations](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/) of making changes within the project. 
+
+As an open source project we don’t make any guarantees about how quickly proposed changes will get looked at. Committers and reviewers may have other pressures on their time, that mean they don’t currently have the bandwidth to look at your proposal. 
```

Comment:
```suggestion
As an open source project we don’t make any guarantees about how quickly proposed changes will get looked at. Committers and reviewers may have other pressures on their time, that mean they don’t currently have the bandwidth to look at your proposal. We do encourage you to discuss your ideas in Slack (in the #development channel or in the relevant SIG channel) so that similarly-minded community members can collaborate with you on the proposal.  
```

Reply:
Looks good :+1: